template/node-arm64/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
Was this intended? Not clear what this has to do with the `store` subcommand.

Reply:
I think these files including NOTES.md have been there when I forked, before sending the PR I merge the upstream but seems that for some reason it didn't remove these files! Anyway, removed all of them.